docs/concatenation.rst

```diff
@@ -3,20 +3,231 @@ Concatenation
 
 .. note::
 
-    This section is currently under construction. There will be more here in the future.
+    The :func:`~anndata.concat` function is marked as experimental for the `0.7` release series, and will supercede the :meth:`AnnData.concatenate() <anndata.AnnData.concatenate>` method in future releases. While the current API is not likely to change much, this gives us a bit of freedom to make sure we've got the arguments and feature set right.
 
-Merging `.uns`
---------------
+With function :func:`~anndata.concat`, `AnnData` objects can be combined via a composition of two operations: concatenation and merging.
```

Comment:
I'd either delete the word "function" or say "With the function". But you're the native speaker.

As this is the first occurrence, here, I'd also do :class:`~anndata.AnnData` instead of just `AnnData`.